tests/suites/test_suite_ssl.function

```diff
@@ -4215,13 +4224,43 @@ void tls13_srv_max_early_data_size(int scenario, int max_early_data_size_arg)
                                written_early_data_size);
                 }
                 break;
+
+            case TEST_EARLY_DATA_SERVER_REJECTS:
+                ret = mbedtls_ssl_handshake(&(server_ep.ssl));
+                /*
+                 * Can be the case if max_early_data_size is smaller then the
+                 * smallest inner content or protected record.
```

Comment:
Actually I think in builds where padding is disabled we should be able to have an exact check here.

Reply:
Yes but it would be rather a test of our client padding algorithm than a test of our server code. Thus I do not really feel the need to improve the testing for that.

Reply:
Nope, I was suggesting this in builds with padding _disabled_ so it wouldn't be a test of the client padding code (which is compiled out then), but of the server's removal of protection overhead (excluding padding because that's the one thing the server can't know). But as discussed yesterday, I think I was overzealous about that, so let's not worry about it for now.